src/app/cmdoptions.h

```diff
@@ -38,7 +38,8 @@ struct CmdOptions {
         ScoreTransposeOptions,
         ForceMode,
         SoundProfile,
-        ExtensionUri
+        ExtensionUri,
+        TransposeOptions
```

Comment:
I think we don't need separate `TransposeOptions` and `ScoreTransposeOptions` keys.

Reply:
Done